index.d.ts

```diff
@@ -5,20 +5,19 @@ import * as SwaggerSchema from 'swagger-schema-official';
 declare namespace fastifySwagger {
   interface FastifySwaggerOptions {
     mode?: 'static' | 'dynamic';
-  }
-
-  interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
-    mode?: 'dynamic';
     /**
      * Overwrite the swagger url end-point
      * @default /documentation
      */
     routePrefix?: string;
     /**
-     *  To expose the documentation api
+     * To expose the documentation api
      * @default false
      */
     exposeRoute?: boolean;
+  }
+
+  interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
```

Comment:
I think `mode?: 'dynamic';` should be kept here. Why did you remove it?

Reply:
Oh, right... Fixed